sdk/include/opentelemetry/sdk/metrics/view/instrument_selector.h

```diff
@@ -27,8 +27,8 @@ class InstrumentSelector
   InstrumentType GetInstrumentType() { return instrument_type_; }
 
 private:
-  opentelemetry::sdk::metrics::InstrumentType instrument_type_;
   std::unique_ptr<opentelemetry::sdk::metrics::Predicate> name_filter_;
+  opentelemetry::sdk::metrics::InstrumentType instrument_type_;
```

Comment:
Is this order change fixing some compiler warning? Probably add comment here if so?

Reply:
yes it is order fixing.  You mean comment in [instrument_selector.h](https://github.com/open-telemetry/opentelemetry-cpp/pull/1312/files/454a2c9861d93330bbec1bbe3098fb16a4e38d79#diff-1df38dbd186fcc5457244dc859258083c012518b7fa8a5de856dae5131780646)?
I'm not sure if we need a comment in the file

Reply:
I think we need to comment only when we have a reason to not to respect the order liker [here](https://github.com/open-telemetry/opentelemetry-cpp/blob/e200c1f183ab1340448b0992e96054f5aa591f46/sdk/include/opentelemetry/sdk/metrics/meter.h#L112).